eng/common/testproxy/transition-scripts/README.md

```diff
@@ -6,16 +6,16 @@ The azure-sdk monorepos are growing quickly due to the presence of recordings. D
 
 The script `generate-assets-json.ps1` will execute the initial migration of your recordings from within a language repo to the [assets repo](https://github.com/Azure/azure-sdk-assets) as well as creating the assets.json file for those assets.
 
-The script is [generate-assets-json.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/testproxy/transition-scripts/generate-assets-json.ps1)
+The script is [generate-assets-json.ps1](https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/testproxy/onboarding/generate-assets-json.ps1)
```

Comment:
Kinda a dumb miss to miss this in my previous eng/common PR. I'll just fix it here.